setup.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from django_ipam import get_version
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+
+def get_install_requires():
+    """
+    parse requirements.txt, ignore links, exclude comments
+    """
+    requirements = []
+    for line in open('requirements.txt').readlines():
+        if line.startswith('#') or line == '' or line.startswith('http') or line.startswith('git'):
+            continue
+        requirements.append(line)
+    return requirements
+
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
```

Comment:
why do you put stuff which is not needed? Please remove this. Always call `python setup.py develop` from the same directory where the file is located.
I repeat: please write only the least amount of code necessary to do the job and nothing more.
We don't like fancy code, because then someone will have to maintain it. Less is more.